REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

nurzatpro opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

cjyuan reviewed (commented):
You are off to a good start. Page looks good and code is well indented.

1. Your page is error free but W3C Markup Validation Service raises some "warnings". Can you address those warnings?

2. One of the acceptance criteria, "The page footer is fixed to the bottom of the viewport", has not yet been satisfied. Can you make the necessary change? (Suggestion: Ask ChatGPT what that requirement means).

3. In your PR description, in the "Changelist" section, can you update the content to briefly describe what you have implemented in this PR branch?
![image](https://github.com/user-attachments/assets/16ed0599-4106-4b4f-a6a9-1bc231ca51de)

4. When I asked ChatGPT to review your code, it gave some useful suggestions which I think can help you improve your implementation. As a practice to get better at using AI tools, try running your code by ChatGPT. You can ask ChatGPT
```
review HTML code:
...        <--- paste your HTML code here
```

Then take a look at the suggestions it gives you, and pick out the ones that actually make sense for your project. It's a great way to learn and improve your work.

nurzatpro commented:
Removed trailing slashes from void elements (e.g., <meta>, <img>) / tag

Ensured footer is fixed to the bottom of the viewport using CSS.

Cleaned up formatting and structure for readability and maintainability.

cjyuan commented:
Changes look good.

Please note that in CYF courses, the recommended way to inform the reviewer of your changes is to do both of the following:
  - Reply to their feedback.
    - In the responses, clarify how each piece of feedback was addressed to demonstrate that you've carefully reviewed the suggestions.
    - Your response may trigger a notification (depending on the reviewer's settings), helping ensure they’re aware of the updates you’ve made. 
  - Replace the "Reviewed" label by a "Needs review" label (which you have done -- great!)
